htdocs/main.php

```diff
@@ -183,8 +183,9 @@ function tplFromRequest($param)
     }
 
     if (isset($caller->page)) {
-        $tpl_data['jsfiles']  = $caller->page->getJSDependencies();
-        $tpl_data['cssfiles'] = $caller->page->getCSSDependencies();
+        $tpl_data['jsfiles']     = $caller->page->getJSDependencies();
+        $tpl_data['cssfiles']    = $caller->page->getCSSDependencies();
+        $tpl_data['breadcrumbs'] = $caller->page->getBreadcrumbs();
```

Comment:
This used to be guarded by "if (!$anonymous) {" and isn't anymore in this change.